pkg/liqoctl/common/cluster.go

```diff
@@ -765,12 +766,16 @@ func mapServiceForCluster(ctx context.Context, ipamClient ipam.IpamClient, ipToB
 // unmapServiceForCluster releases a mapped local ip address previously mapped for given remote cluster.
 func unmapServiceForCluster(ctx context.Context, ipamClient ipam.IpamClient, ipToBeUnmapped string,
 	remoteCluster *discoveryv1alpha1.ClusterIdentity) error {
+	var err error
+
 	unMapRequest := &ipam.UnmapRequest{
 		ClusterID: remoteCluster.ClusterID,
 		Ip:        ipToBeUnmapped,
 	}
 
-	_, err := ipamClient.UnmapEndpointIP(ctx, unMapRequest)
+	if _, err = ipamClient.UnmapEndpointIP(ctx, unMapRequest); err != nil && strings.Contains(err.Error(), "must be initialized first") {
```

Comment:
Would personally add a comment to point out why this might happen and an example of the full error message (as in the PR description), for easier retrieval in the future.